From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: IBM Codestream linux-2.6.16 october2005, patch base-10

[patch] dasd: remove experimental flag from dasd diag.

The dasd diag discipline has been tested on 64 bit and is no longer
experimental.
Acked-by: Hannes Reinecke <hare@suse.de>

---

 drivers/s390/block/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -urpN linux-2.6/drivers/s390/block/Kconfig linux-2.6-patched/drivers/s390/block/Kconfig
--- linux-2.6/drivers/s390/block/Kconfig	2006-03-20 06:53:29.000000000 +0100
+++ linux-2.6-patched/drivers/s390/block/Kconfig	2006-03-22 11:46:50.000000000 +0100
@@ -49,7 +49,7 @@ config DASD_FBA
 
 config DASD_DIAG
 	tristate "Support for DIAG access to Disks"
-	depends on DASD && ( 64BIT = 'n' || EXPERIMENTAL)
+	depends on DASD
 	help
 	  Select this option if you want to use Diagnose250 command to access
 	  Disks under VM.  If you are not running under VM or unsure what it is,
